### PR TITLE
Updated plugin XML

### LINKED ISSUE #273: Plugin XML out of date
### Actual behaviour
The XML of the plugin info is out dated, the repo slug changed, also there is necessary to update the English and french description of the plugin, translate it in some more languages (Spanish at least).

### Steps to reproduce the behaviour

1. Read the plugin.xml on the root folder of the project.

### PR DESCRIPTION
**Pull request by DIOHz0r**
## Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Conventional Commit](http://conventionalcommits.org/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Fixes #273 

## What is the new behavior?
Updated information

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
       <short>
          <fr>Flyve MDM est une solution de gestion et d'administation d'appareils mobiles.</fr>
          <en>Flyve MDM is a mobile devices management and administration solution.</en>
+         <en>Flyve MDM es una solución administrativa y de gestión de dispositivos móviles.</en>
```
Comment (Naylin15) [inline plugin.xml]: Hi @DIOHz0r the tag should be <es> for the spanish translation

File: plugin.xml [hunk 2]
```diff
@@ -25,25 +26,39 @@ Fonctionnalités
...
-1. Mobile devies enrolment
+1. Mobile devices enrolment
 2. File deployment on mobile devices
 3. Application deployment on mobile devices
 4. Configuration of mobile devices (password, connectivity, lock)
 5. Geolocation of the mobile devices
 6. Emergency remote wipe of mobile devices
          </en>
+         <es>
+FlyveMDM es una solución administrativa y de administración de dispositivos móviles. El plugin se ejecuta en una instancia dedicada de GLPI.
```
Comment (Naylin15) [inline plugin.xml]: I think it is better the translation in line 10,

Flyve MDM es una solución administrativa y de ***gestión*** de dispositivos móviles.

File: plugin.xml [hunk 3]
```diff
@@ -64,12 +79,21 @@ Features
          <tag>smartphone</tag>
          <tag>configuration</tag>
          <tag>android</tag>
+         <tag>ios</tag>
       </fr>
       <en>
          <tag>inventaire</tag>
```
Comment (Naylin15) [inline plugin.xml]: Inventaire is the french translation, it should be inventory

In line 78 is the opposite

File: plugin.xml [hunk 4]
```diff
@@ -25,25 +26,39 @@ Fonctionnalités
...
 2. File deployment on mobile devices
 3. Application deployment on mobile devices
 4. Configuration of mobile devices (password, connectivity, lock)
 5. Geolocation of the mobile devices
 6. Emergency remote wipe of mobile devices
          </en>
+         <es>
+FlyveMDM es una solución administrativa y de administración de dispositivos móviles. El plugin se ejecuta en una instancia dedicada de GLPI.
+
+Para mas informacion, por favor visite https://flyve-mdm.com/
```
Comment (Naylin15) [inline plugin.xml]: Missing accent mark in información

File: plugin.xml [hunk 5]
```diff
@@ -25,25 +26,39 @@ Fonctionnalités
...
+
+Caracteristicas
```
Comment (Naylin15) [inline plugin.xml]: Missing accent mark in Características

File: plugin.xml [hunk 6]
```diff
@@ -7,6 +7,7 @@
       <short>
          <fr>Flyve MDM est une solution de gestion et d'administation d'appareils mobiles.</fr>
```
Comment (Naylin15) [inline plugin.xml]: Hi, @DIOHz0r 

It is missing the r in et *d'administration*

**Comment by ajsb85** [comment]
@DIOHz0r If the status checks from Travis CI, keep the title as WIP

Travis CI error:

```
The command "vendor/bin/atoum -ft -bf tests/bootstrap.php -d tests/suite-integration --nccfc CommonTreeDropdown CommonDropdown CommonDBTM CommonGLPI" exited with 255.
```

**Comment by DIOHz0r** [comment]
Travis is not failing anymore, please approve.

**Pull request merged**